Sources/Testing/Traits/TimeLimitTrait.swift

```diff
@@ -54,35 +52,34 @@ public struct TimeLimitTrait: TestTrait, SuiteTrait {
 
 @available(_clockAPI, *)
 extension Trait where Self == TimeLimitTrait {
-  /// Construct a time limit trait that causes a test to time out if it runs for
-  /// too long.
+  /// Constructs a time limit trait that causes a test to time out if it runs
+  /// for too long.
   ///
   /// - Parameters:
   ///   - timeLimit: The maximum amount of time the test may run for.
   ///
   /// - Returns: An instance of ``TimeLimitTrait``.
   ///
   /// Test timeouts do not support high-precision, arbitrarily short durations
-  /// due to variability in testing environments. The time limit must be at
-  /// least one minute, and can only be expressed in increments of one minute.
+  /// due to variability in testing environments. You express the duration in
+  /// minutes, with a minimum duration of one minute.
   ///
-  /// When this trait is associated with a test, that test must complete within
-  /// a time limit of, at most, `timeLimit`. If the test runs longer, an issue
-  /// of kind ``Issue/Kind/timeLimitExceeded(timeLimitComponents:)`` is
-  /// recorded. This timeout is treated as a test failure.
+  /// When you associate this trait with a test, that test must complete within
+  /// a time limit of, at most, `timeLimit`. If the test runs longer, the
+  /// testing library records a
+  /// ``Issue/Kind/timeLimitExceeded(timeLimitComponents:)`` issue, which it
+  /// treats as a test failure.
   ///
-  /// The time limit amount specified by `timeLimit` may be reduced if the
-  /// testing library is configured to enforce a maximum per-test limit. When
-  /// such a maximum is set, the effective time limit of the test this trait is
-  /// applied to will be the lesser of `timeLimit` and that maximum. This is a
-  /// policy which may be configured on a global basis by the tool responsible
-  /// for launching the test process. Refer to that tool's documentation for
-  /// more details.
+  /// The testing library can use a shorter time limit than that specified by
+  /// `timeLimit` may be reduced if it's configured to enforce a maximum
+  /// per-test limit. When you configure a maximum per-test limit, the time
```

Comment:
I think the first sentence of this paragraph was only partly reworded and needs another look

Reply:
OK had another go at this paragraph, see what you think.